arm-service-map/2015-11-01-preview/swagger/arm-service-map.json

```diff
@@ -1718,6 +1718,17 @@
               "format": "date-time",
               "description": "UTC date and time when the process started"
             },
+            "role": {
+              "type": "string",
```

Comment:
From the looks of it the property "role" is a part of model "Process" which is a part of response models only, this shouldn't be a breaking change.
tldr; LGTM